app.json

```diff
@@ -0,0 +1,24 @@
+{
+    "name": "vali-ah-backend",
+    "description": "1kbIdeas: The one-stop shop for all the self-help ideas you need",
+    "scripts": {},
+    "env": {
+      "NODE_ENV": {
+        "required": true
+      }
+    },
+    "formation": {
+      "web": {
+        "quantity": 1
+      }
+    },
+    "addons": [
+      "heroku-postgresql"
+    ],
+    "buildpacks": [
+      {
+        "url": "heroku/nodejs"
+      }
+    ],
+    "stack": "heroku-18"
+  }
```

Comment:
Add new line

Reply:
is not a new line issue, the closing tag is not properly indented.